lib/modules/manager/ocb/readme.md

```diff
@@ -0,0 +1,21 @@
+Renovate uses this manager to update dependencies defined in the build definitions for the [OpenTelemetry Collector Builder (ocb)](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder).
+
+The `ocb` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
```

Comment:
```suggestion
By default, the `ocb` manager has no `fileMatch` patterns.
This means you must set a `fileMatch` pattern for the `ocb` manager, in order for Renovate to update your `ocb` files.
Here's a configuration example:
```

How about something like this? We should explain that the user must do something first. And put our point positively, so from `it won't match any files until you ...` to: `This means you must set...`